development/jobguard/README.md

```diff
@@ -11,24 +11,30 @@ The main purpose of Job Guard is to delay running integration jobs that depend o
 
 To run the application, use this command:
 
-```bash
-COMMIT_SHA={commit_sha} PROW_CONFIG_FILE={prow_config_file} PROW_JOBS_DIRECTORY={prow_jobs_directory} go run cmd/main.go
+|Flag|Required|Description|
```

Comment:
Please add this section name here. I'd go with `Flags`, but [prowjobparser](https://github.com/kyma-project/test-infra/tree/cef5cedd57404c2aa971662f9bff926ad3b90285/development/prowjobparser) already uses `CLI parameters`. 

```suggestion
### CLI parameters

|Flag|Required|Description|
```